text/0000-trait_stable_vtable.md

```diff
@@ -184,6 +184,8 @@ Additionally, something similar to COM could be used as the stablized vtable lay
 
 As mentioned, both the C++ Itanium ABI and COM are examples of a "stablized" virtual dispatch. Both specify the layout of virtual dispatch tables, and how they interact with virtual calls. COM-like structs have specifically been used for cross-language "virtual" dispatch, and have been used in a variety of applications, including the Java Native Interface.
 
+The [abi\_stable](https://crates.io/crates/abi_stable) crate provides a semi-stablized vtable for Rust-Rust ABI Compatibility. The compatibility of this vtable with the proposed layout is being investigated. 
+
```

Comment:
Author of abi_stable here.
The way that the vtable is implemented is incompatible, it has entries for methods of a few standard library traits, which are unused if the trait doesn't have them as supertraits. Also, the size and alignment of the type aren't stored there, only the destructor.

Reply:
Indeed, I looked after adding that. I'll add a note saying its incompatible. I doubt adding in compatibility here would fall within the goals of the rfc, as the standard library traits may not make sense from a foreign-language perspective. For example, I believe that a valid supertrait is the Iterator trait, which may not exist in a foreign-language with similar functionality to this rfc, or may exist in a different form.